lib/ledger_sync/util/hash_helpers.rb

```diff
@@ -8,6 +8,10 @@ module HashHelpers
       def deep_symbolize_keys(hash)
         JSON.parse(JSON[hash], symbolize_names: true)
       end
+
+      def deep_stringify_keys(hash)
```

Comment:
This will cause similar issue like `deep_symbolize_keys`. it will serialize to string ruby objects like Date, etc. We should update it similar like we did in https://github.com/LedgerSync/ledger_sync/pull/7/files#diff-3d4ff3d9c66393d3f07187d12ff8eec2R11-R24.